keras_cv/__init__.py

```diff
@@ -13,6 +13,13 @@
 # limitations under the License.
 
 # isort:off
+try:
+    # When using torch and tensorflow, torch needs to be imported first,
+    # otherwise it will segfault upon import.
+    import torch
```

Comment:
```suggestion
    import torch
    
    del torch
```

We probably don't want torch lingering in the public API in case someone does `from keras_core import *`

Reply:
Good call -- done